app/views/play/level/ControlBarView.coffee

```diff
@@ -188,6 +193,11 @@ module.exports = class ControlBarView extends CocoView
     @options.hintsState.set('hidden', not @options.hintsState.get('hidden'))
     window.tracker?.trackEvent 'Hints Clicked', category: 'Students', levelSlug: @levelSlug, hintCount: @options.hintsState.get('hints')?.length ? 0
 
+  onOverallStatusChanged: (e) ->
+    return if e?.overallStatus == @lastOverallStatus 
+    @lastOverallStatus = e.overallStatus
+    @render()
```

Comment:
this is okay. while also can use `renderSelectors('.buttons-area')` which refresh less areas.

Reply:
I tried changing it to
`@renderSelectors('.buttons-area')`, but it seems to cause the buttons area to not render at all
![image](https://github.com/user-attachments/assets/469cac2d-f98b-442d-a46a-ce34a24b89a5)